pkg/cache/sql/integration_test.go

```diff
@@ -0,0 +1,322 @@
+package sql
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/rancher/lasso/pkg/cache/sql/informer"
+	"github.com/rancher/lasso/pkg/cache/sql/informer/factory"
+	"github.com/rancher/lasso/pkg/cache/sql/partition"
+	"github.com/stretchr/testify/suite"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/cache"
+	"sigs.k8s.io/controller-runtime/pkg/envtest"
+)
+
+const testNamespace = "sql-test"
+
+var defaultPartition = partition.Partition{
+	All: true,
+}
+
+type IntegrationSuite struct {
+	suite.Suite
+	testEnv   envtest.Environment
+	clientset kubernetes.Clientset
+	restCfg   rest.Config
+}
+
+func (i *IntegrationSuite) SetupSuite() {
+	i.testEnv = envtest.Environment{}
+	restCfg, err := i.testEnv.Start()
+	i.Require().NoError(err, "error when staring env test - this is likely because setup-envtest wasn't done. Check the README for more information")
```

Comment:
```suggestion
	i.Require().NoError(err, "error when starting env test - this is likely because setup-envtest wasn't done. Check the README for more information")
```